### PR TITLE
[WIP] IPU: Firewalld actors for el9 to el10

### PR DESCRIPTION
**Pull request by erig0**
The main actor, FirewalldIptablesModules, will install kernel-modules-extra if it's determined that firewalld configuration depends upon iptables.

### CODE REVIEW

**Comment by erig0** [comment]
review please @oamg/developers

File: repos/system_upgrade/el9toel10/models/firewalldglobalconfig.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    flushallonreload = fields.Boolean(default=True)
+    reloadpolicy = fields.String(default='INPUT:DROP,FORWARD:DROP,OUTPUT:DROP')
+    rfc3964_ipv4 = fields.Boolean(default=True)
+    nftablesflowtable = fields.String(default='off')
+    nftablescounters = fields.Boolean(default=False)
+    nftablestableowner = fields.Boolean(default=False)
+
+    # These have been removed in RHEL-10.
+    #
+    allowzonedrifting = fields.Boolean(default=False)
```
Comment (pirat89) [inline repos/system_upgrade/el9toel10/models/firewalldglobalconfig.py:30]: we will go through this PR most likely after we finish work on CTC1 builds (~2w). however, `allowzonedrifting` has been already removed in rhel9 - at least based on existing actors for IPU 8 -> 9.

Comment (erig0) [reply repos/system_upgrade/el9toel10/models/firewalldglobalconfig.py:30]: It's not removed. Just deprecated and ignored. I'll change the comment.

**Comment by erig0** [comment]
Please hold off on merging this PR. We may not need these actors after all.

Ref: https://issues.redhat.com/browse/RHEL-65224

**Comment by erig0** [comment]
Closing as per above. These actors are not necessary as this issue will be handled at the RPM level.

**Pull request closed without merging**